src/Display/Modal/Modal.test.tsx

```diff
@@ -188,29 +188,42 @@ describe('displays the correct size', () => {
   });
 });
 
-it('should not show the header when hideHeader is true', () => {
-  const { getByTestId } = render(
-    <Modal
-      hideHeader
-      title={props.title}
-      isVisible={true}
-      onClose={props.onClose}
-    >
-      <p>{props.content}</p>
-    </Modal>
-  );
-  const modalContainer = getByTestId('modal-container');
-  expect(modalContainer).not.toHaveTextContent(props.title);
-});
+describe('<Modal /> UI', () => {
+  it('should not show the header when hideHeader is true', () => {
+    const { getByTestId } = render(
+      <Modal
+        hideHeader
+        title={props.title}
+        isVisible={true}
+        onClose={props.onClose}
+      >
+        <p>{props.content}</p>
+      </Modal>
+    );
+    const modalContainer = getByTestId('modal-container');
+    expect(modalContainer).not.toHaveTextContent(props.title);
+  });
 
-it('should center the Modal vertically when centering is true', () => {
-  const { getByTestId } = render(
-    <Modal centering isVisible={true} onClose={props.onClose}>
-      <p>{props.content}</p>
-    </Modal>
-  );
-  const modalContainer = getByTestId('modal-container');
-  expect(modalContainer).toHaveStyle('align-items: center;');
+  it('should center the Modal vertically when centering is true', () => {
+    const { getByTestId } = render(
+      <Modal centering isVisible={true} onClose={props.onClose}>
+        <p>{props.content}</p>
+      </Modal>
+    );
+    const modalContainer = getByTestId('modal-container');
+    expect(modalContainer).toHaveStyle('align-items: center;');
+  });
+
+  it('should make the Modal cover the entire screen', () => {
+    const { getByTestId } = render(
+      <Modal fullscreen isVisible={true} onClose={props.onClose}>
+        <p>{props.content}</p>
+      </Modal>
+    );
+    const modalContentArea = getByTestId('dialog');
+    expect(modalContentArea).toHaveStyle('width: 100vw;');
+    expect(modalContentArea).toHaveStyle('height: 100vh;');
```

Comment:
Just follow the above test case to test style. Please let me know if there is a better way to test the fullscreen UI.

Reply:
I think I'd prefer a snapshot test. But it's okay to go with consistency here.